hw_diag/templates/diagnostics_page.html

```diff
@@ -102,27 +76,35 @@ <h3 class="text-center mb-4">Diagnostics Breakdown</h3>
                   <td class="text-right">{{ diagnostics.FRIENDLY }}</td>
                 </tr>
                 <tr>
-                  <td>Miner Connected To Blockchain</td>
-                  {% if diagnostics.MC %}
-                    <td class="text-success text-right">True</td>
+                  <td>Connected Validator</td>
+                  {% if diagnostics.validator_uri %}
+                    <td class="text-success text-right">{{ diagnostics.validator_uri }} </td>
                   {% else %}
-                    <td class="text-warning text-right">False</td>
+                    <td class="text-warning text-right">Disconnected</td>
                   {% endif %}
                 </tr>
+                <tr>
+                    <td>Validator Block Height</td>
```

Comment:
open to better UI string for field names. These feel too verbose.

Reply:
I'm OK with these but I get your point.